Add CSS font-size property (by kettlebell)

epost opened the pull request:
This enhancement was made by kettlebell: https://github.com/kettlebell/purescript-halogen-svg/commit/d6d96caa5f43abfedde511dc7a30b2f38558f865.

epost commented:
FYI @kettlebell, I made a PR out of your contribution.

kettlebell commented:
Hi @epost, thanks for arranging the pull request. I've recently returned to my repo and updated it to use Spago and Halogen 5 (wasn't too bad as you can see [here](https://github.com/kettlebell/purescript-halogen-svg/commit/cb3cb5bb7bdcad7651a4c9b1cdf63665cdad6f01))

Also it'd be great to flesh out some more of the SVG properties.

epost commented:
@kettlebell That's good to hear! I'm doing this in the hopes of converging on a repo that can be maintained, because we're using this at Statebox and I've added [a fair number of features](https://github.com/epost/purescript-halogen-svg/commit/fa7a0c155ea875f712e8735febcac40abcc6ec02), including animation support. Here is a WIP document describing the current state of affairs: https://hackmd.io/lYvFcjb7STe2YzIet9sM7w?both. I've added the changes you just mentioned to the document.

@kwohlfahrt Would you consider making some of us contributors, so we can collaborate on maintenance tasks?

rskew commented:
@epost I've started using this repo and I've added a couple of small things here https://github.com/kwohlfahrt/purescript-halogen-svg/pull/14 (it looks like I've duplicated the marker functionality, woops).

If you have another repo for merging various changes I'll happily create a PR there :)

epost commented:
Hey @rskew, that is cool stuff! :D And a coincidence too; Sjoerd Visscher just started working on this yesterday at https://github.com/statebox/purescript-halogen-svg. Would be great if you could make a PR.

rskew commented:
Cheers @epost. I'll aim to make a PR soon, I'll have to update my parent project from Halogen 4 to 5 first.

kwohlfahrt commented:
@epost @rskew - sorry for the long silence. As you guys can see, I'm no longer doing any purescript development so I don't have any personal use for this library. Would you prefer to be collaborators here, or alternatively I can replace it with a placeholder redirecting to another repo.

rskew commented:
@kwohlfahrt all good :) I'm up for whatever.

srghma commented:
@kwohlfahrt hi, could you add me as a maintainer, I want to merge all currently open PRs and update to the latest halogen

I want to use this repo in https://github.com/srghma/purescript-halogen-nextjs
